docs/guidance/ingress.md

```diff
@@ -201,3 +201,12 @@ kubectl describe ingress raycluster-ingress-head-ingress
 #        [Note] The forward slash at the end of the address is necessary. `<ip>/raycluster-ingress`
 #               will report "404 Not Found".
 ```
+
```

Comment:
Maybe we just need to add a comment in Step 2 that says to execute the kubectl wait command after the Pod for the NGINX ingress controller is created or just `sleep 10`. This is an expected behavior for `kubectl wait`.

Reply:
The commands are copied from [here](https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx).